Jenkinsfile.central

```diff
@@ -0,0 +1,73 @@
+#!groovy
+
+node{
+   properties(
+       [parameters(
+           [string(defaultValue: 'develop', description: 'The SHA for the commit you wish to deploy.', name: 'GIT_COMMIT', trim: false),
+            string(defaultValue: '', description: 'The tag of the current release.', name: 'RELEASE_TAG', trim: false)]
+        )]
+    )
+    
+    try{
+        env.AWS_DEFAULT_REGION = "us-east-1"
+
+        // Checkout the proper revision into the workspace.
+        stage('checkout') {
```

Comment:
I couldn't get the workspace to clean properly because the sbt container image is writing files as uid 0.

Reply:
Hm. I'd expect this to apply to the other builds too, since they're likely all writing to the host file system as `root`.